### DISCUSSION TITLE
Apply specific summary to IgxGrid (IgbGrid)

### DISCUSSION THREAD

**Discussion by agoldenbaum** [IgniteUI/igniteui-angular#14112] [Q&A] [answered]
This is really surrounding a question for Blazor from a support case, but I am not seeing a way to do this in the IgxGrid in Angular either, so I'll ask it here.

Is there a way to apply a specific summary to a numeric IgxColumn without the need to do a custom summary? Currently, it appears that the behavior is that if you have an IgxColumn with a numeric data type and set **hasSummary** to true, it will show summaries for Count, Min, Max, Sum and Avg, as shown here: https://www.infragistics.com/products/ignite-ui-angular/angular/components/grid/summaries.

Using the example above, how would I, for example, just show "Sum" for the UnitPrice column? Is there something to configure this, like a property or event, or is a custom summary necessary?

- Andrew

**Comment by ddaribo** [comment, marked as answer]
Hi @agoldenbaum,

Yes, the approach to modify the list of summary results is to create a custom summary class to assign to the column’s [summaries](https://www.infragistics.com/products/ignite-ui-angular/docs/typescript/latest/classes/IgxColumnComponent.html#summaries) property and modify what would be included in the corresponding summary operand’s results. 
The docs demos show some [examples](https://www.infragistics.com/products/ignite-ui-angular/angular/components/grid/summaries#demo) of this, however, I am afraid this is the only exposed API to achieve this currently.

**Comment by agoldenbaum** [comment]
Thanks for the update, @ddaribo. I have created a feature request for this here: https://github.com/IgniteUI/igniteui-angular/issues/14120
